### DISCUSSION TITLE
Switching from Mirador2 to Mirador3

### DISCUSSION THREAD

**Discussion by agniramadani** [ProjectMirador/mirador#3649] [General]
I have an application running on Ruby on Rails, [Mirador2](https://github.com/Teifun2/mirador_rails) is used, it works fine, and all the annotations are showing.

### **Issue**:
I have created an app using ReactJs and used Mirador3 with the same [manifest](https://db.d-scribes.philhist.unibas.ch/api/version1/iiif/writer1/manifest) made from Ruby on Rails, but it shows 0 annotations.

### **Goal:**
Show the annotations in ReactJs app using Mirador3 with the mentioned manifest.

Any recommendation?

**Comment by lutzhelm** [comment]
Sorry for the late answer. It seems like your example manifest does not link to any annotations by itself - these would be found in the `otherContent` property of the respective Canvas. My guess would be that your Mirador 2 config contains a configured annotation store.
